dockerfile.lint.cloudbuild.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+- id: 'Lint Dockerfile'
+  name: 'hadolint/hadolint:latest-alpine' # https://hub.docker.com/r/hadolint/hadolint
+  entrypoint: '/bin/sh'
+  args:
+  - -c
+  - | # Recursively find dockerfiles (not in testing dir)
+    find . -name '*Dockerfile' -not -path "./testing/*" -exec hadolint {} \;
+    find . -name '*Dockerfile' -not -path "./testing/*" \
+      -exec hadolint {} \;| grep -q 'Dockerfile'
```

Comment:
I read this as us running the linter twice, why is that needed?

Reply:
It runs once to print the results, the second time silently greps if there are any errors in order to manually throw the error. This is needed because sometimes the output of the commands isn't triggering a build failure.